src/components/Charts.js

```diff
@@ -0,0 +1,83 @@
+import { useSelector } from 'react-redux';
+import { 
+  Sparklines, 
+  SparklinesLine, 
+  SparklinesReferenceLine 
+} from 'react-sparklines';
+
+
+const Charts = () => {
+  const forecast = useSelector((state) => state.forecast)
+  const dataAvg = (data) => {
+    const findAvg = data.reduce((a, b) => a + b, 0);
+    return Math.round( findAvg / data.length);
+  }
+
+  return forecast.map((search, id) => (
+  
+    <table key={id}>
+      <tbody>
+        <tr>
+          <td>
+            <h3>{search.data.city.name}</h3>
+          </td>
+
+          <td>
+            <Sparklines 
+              svgWidth={200}
+              svgHeight={150}
+              data={search.data.list.map((chart) => {
+                return chart.main.temp;
+              })}>
+              <SparklinesLine 
+                color="green" />
+              <SparklinesReferenceLine 
+                type="avg" />
```

Comment:
I'm not sure these are strictly necessary (they make it so there's more to take in when reading the code) but not necessarily a detractor